scripts/ci-apidiff.sh

```diff
@@ -20,9 +20,12 @@ set -o pipefail
 
 REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 
-APIDIFF="${REPO_ROOT}/hack/tools/bin/go-apidiff"
-
-cd "${REPO_ROOT}" && make "${APIDIFF##*/}"
-echo "*** Running go-apidiff ***"
-
-${APIDIFF} "${PULL_BASE_SHA}" --print-compatible
\ No newline at end of file
+cd "${REPO_ROOT}"
+APICHANGE=$(git --no-pager diff --name-only FETCH_HEAD)
+if echo "${APICHANGE}" | grep "api/" 
```

Comment:
I think if we grep'd for `"/api/"` (both forward and trailing slashes) we'd be a bit more defensive (in a good way) going forward. For example, if someone created a subdirectory somewhere with a substring suffix of "capi" this would match, which we wouldn't want.

Thanks for doing this!

Reply:
I don't think this change with the leading slash will work correctly, though:

```shell
% git --no-pager diff --name-only FETCH_HEAD | grep "/api/"
% echo dummy >> api/v1beta1/doc.go                         
% git --no-pager diff --name-only FETCH_HEAD | grep "/api/"
% git --no-pager diff --name-only FETCH_HEAD | grep "api/" 
api/v1beta1/doc.go

```